app/views/layouts/card_wide.html.slim

```diff
@@ -1,3 +1,3 @@
-- card_cls 'card-wide'
+- card_cls 'card-wide sm-px2 px0'
```

Comment:
so i think what we are going to have to do is add a bunch of `px2 sm-px0` on basically every div inside of the container except the ones that we have full width in order to give some padding on mobile:

![screen shot 2017-04-07 at 3 36 22 pm](https://cloud.githubusercontent.com/assets/1178494/24816788/1b4dc868-1ba8-11e7-8421-960f8213a390.png)